src/main/java/com/algorand/algosdk/v2/client/model/AccountsErrorResponse.java

```diff
@@ -0,0 +1,65 @@
+package com.algorand.algosdk.v2.client.model;
```

Comment:
As discussed, I don't think this type would be properly used with the current response handler.